scripts/globals/weaponskills.lua

```diff
@@ -117,6 +117,10 @@ function doPhysicalWeaponskill(attacker, target, wsID, tp, primary, action, taCh
         if (attacker:hasStatusEffect(EFFECT_INNIN) and attacker:isBehind(target, 23)) then -- Innin acc boost attacker is behind target
             nativecrit = nativecrit + attacker:getStatusEffect(EFFECT_INNIN):getPower();
         end
+        
+        if (attacker:hasStatusEffect(EFFECT_ASSASSINS_CHARGE)) then
+            nativecrit = nativecrit + attacker:getStatusEffect(EFFECT_ASSASSINS_CHARGE):getSubPower()/100;
```

Comment:
doesn't assassin's charge already add the crithitrate mod?

Reply:
I'll double check that, but I believe you're correct.

Reply:
Yep. That's how that works. Gutting this.